app/date-select/controller.js

```diff
@@ -0,0 +1,34 @@
+const FormWizardController = require('../../common/controllers/form-wizard')
+const { dateRegex } = require('../../common/helpers/url')
+
+const DateRegExp = new RegExp('/' + dateRegex)
+
+class DateSelectController extends FormWizardController {
+  get(req, res, next) {
+    if (req.query.referrer) {
+      req.sessionModel.reset()
```

Comment:
I think this might cause issues with users with multiple tabs.

Scenario: 
I start entering a move, allocation or escort record
I open a new tab to view a previous move I want to copy data from
I use the jump to date feature to get to the move I want
I go back to my previous form, and it has now been reset.

Reply:
Maybe we could set this up as a different journey instead?

Reply:
As discussed the errors get stashed on the sessionModel for that specific wizard config - it doesn't effect the other, er, journeys